addons/loyalty/views/loyalty_program_views.xml

```diff
@@ -48,14 +48,14 @@
                                 <p class="text-muted" attrs="{'invisible': [('program_type', '!=', 'coupons')]}" colspan="2">
                                     Generate &amp; share coupon codes manually. It can be used in eCommerce, Point of Sale or regular orders to claim the Reward. You can define constraints on its usage through conditional rule.
                                     <div groups="base.group_no_one">
-                                        When generating coupon, you can define a specific points value that can be exchanged for rewards. 
+                                        When generating coupon, you can define a specific points value that can be exchanged for rewards.
                                     </div>
                                 </p>
                                 <p class="text-muted" attrs="{'invisible': [('program_type', '!=', 'loyalty')]}" colspan="2">
                                     When customers make an order, they accumulate points they can exchange for rewards on the current order or on a future one.
                                 </p>
                                 <p class="text-muted" attrs="{'invisible': [('program_type', '!=', 'promotion')]}" colspan="2">
-                                    Set up conditional rules on the order that will give access to rewards for customers                                            
+                                    Set up conditional rules on the order that will give access to rewards for customers
```

Comment:
We can do these changes in another PR if you wish but this is not the place to do so.